pandas/tests/io/test_pickle.py

```diff
@@ -155,28 +156,43 @@ def test_pickles(current_pickle_data, legacy_pickle):
         compare(current_pickle_data, legacy_pickle, version)
 
 
-def test_round_trip_current(current_pickle_data):
-    def python_pickler(obj, path):
-        with open(path, "wb") as fh:
-            pickle.dump(obj, fh, protocol=-1)
+def python_pickler(obj, path):
+    with open(path, "wb") as fh:
+        pickle.dump(obj, fh, protocol=-1)
 
-    def python_unpickler(path):
-        with open(path, "rb") as fh:
-            fh.seek(0)
-            return pickle.load(fh)
 
+def python_unpickler(path):
+    with open(path, "rb") as fh:
+        fh.seek(0)
+        return pickle.load(fh)
+
+
+@pytest.mark.parametrize(
+    "pickle_writer",
+    [
+        pytest.param(python_pickler, id="python"),
```

Comment:
you can omit the `pytest.param` for all of these, except the xfailed one (if you are ok with the id that is generated) i believe

Reply:
but this is pretty clear too, so up to you if it works

Reply:
IMHO it looks less clear without:
```
pandas/tests/io/test_pickle.py::test_round_trip_current[python_pickler] PASSED                                                                                                               [ 20%]
pandas/tests/io/test_pickle.py::test_round_trip_current[to_pickle] PASSED                                                                                                                    [ 40%]
pandas/tests/io/test_pickle.py::test_round_trip_current[pickle_writer2] PASSED                                                                                                               [ 60%]
pandas/tests/io/test_pickle.py::test_round_trip_current[pickle_writer3] PASSED                                                                                                               [ 80%]
pandas/tests/io/test_pickle.py::test_round_trip_current[pandas_proto_5] PASSED  
```
- so I would prefer to leave it with custom `id`s